test/xskbench/xskbench.c

```diff
@@ -1669,8 +1701,13 @@ ParseArgs(
         } else if (!strcmp(argv[i], "-v")) {
             verbose = TRUE;
         } else if (!_stricmp(argv[i], "-lp")) {
+#if defined(_KERNEL_MODE)
+            printf_error("Large pages not supported in kernel mode\n");
```

Comment:
Can't kernel allocate any page size it wants?

Reply:
Yeah, I think allocations that are multiples of 2MB (on amd64) are automatically allocated as large pages, if possible. I'm not sure the exact logic within MM.

Reply:
Thanks for enlightening me. I've removed this artificial error